webapp/channels/src/components/post_edit_history/edited_post_item/__snapshots__/edited_post_item.test.tsx.snap

```diff
@@ -85,24 +85,10 @@ exports[`components/post_edit_history/edited_post_item should match snapshot 1`]
             />
           </span>
         </div>
-        <OverlayTrigger
-          defaultOverlayShown={false}
-          delayShow={400}
-          overlay={
-            <Tooltip
-              className="hidden-xs"
-              id="editPostRestoreTooltip"
-            >
-              Restore this version
-            </Tooltip>
-          }
+        <WithTooltip
+          id="editPostRestoreTooltip"
           placement="left"
-          trigger={
-            Array [
-              "hover",
-              "focus",
-            ]
-          }
+          title="Restore this version"
```

Comment:
Should we use `Intl` `formatMessage` for the title?

Reply:
This is snapshot it does used i18n

Reply:
Aah, my bad.